gp-nested-forms/gpnf-auto-attach-child-files.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Gravity Perks // Nested Forms // Auto-attach Uploaded Files from Child to Parent Notifications
+ * https://gravitywiz.com/documentation/gravity-forms-nested-forms/
+ *
+ * This snippet automatically attaches files uploaded onto the child form to the parent form notifications.
+ *
+ * Plugin Name:  GP Nested Forms - Auto-attach Uploaded Files from Child to Parent Notifications
+ * Plugin URI:   http://gravitywiz.com/documentation/gravity-forms-nested-forms/
+ * Description:  Auto-attach Uploaded Files from Child to Parent Notifications
+ * Author:       Gravity Wiz
+ * Version:      0.1
+ * Author URI:   https://gravitywiz.com/
+ */
+add_filter( 'gform_notification', function( $notification, $form, $entry ) {
+
+	if( ! class_exists( 'GPNF_Entry' ) ) {
+		return $notification;
+	}
+
+	$upload_fields = GFCommon::get_fields_by_type( $form, array( 'fileupload' ) );
+
+	// If parent form has upload fields, rely on the notification's Attachments setting.
+	if( ! empty( $upload_fields ) ) {
+		if ( ! rgar( $notification, 'enableAttachments', false ) ) {
+			return $notification;
+		}
+	}
+	// Otherwise, rely on a manually defined array of notification IDs.
+	else {
+		$notification_ids = array( '5daaedb49dc32', '5dbce25cc21c2' );
```

Comment:
Let's add an inline instruction here to update these to the notification IDs to which this should apply. Notification IDs can be retrieved from the `nid` parameter when editing a notification. Confirm that's still true. 😅